### PR TITLE
Appkit for Xcode Beta 2

### PR DESCRIPTION
**Pull request by chamons**
I did some of these by hand, so please double check for silly mistakes.

### CODE REVIEW

File: src/AppKit/Enums.cs [hunk 1]
```diff
@@ -2582,4 +2586,31 @@ public enum NSCloudKitSharingServiceOptions : nuint
 		AllowReadWrite = 1 << 5
 	}
 
+	[Mac (10, 12)]
+	[Native]
+	public enum NSDisplayGamut : nint
+	{
+		SRGB = 1,
```
Comment (spouliot) [inline src/AppKit/Enums.cs]: `Srgb` -> more than 2 letters acronym rule (it's also what's used on XI).

Comment (chamons) [inline src/AppKit/Enums.cs]: Dang it. I know better that missing that. :(

File: src/appkit.cs [hunk 2]
```diff
@@ -19090,6 +19130,10 @@ public interface NSTitlebarAccessoryViewController : NSAnimationDelegate, NSAnim
 
 		[Export ("viewDidDisappear")]
 		void ViewDidDisappear ();
+
+		[Mac (10,12)]
+		[Export ("hidden")]
+		bool IsHidden { [Bind ("isHidden")]get; set; }
```
Comment (spouliot) [inline src/appkit.cs]: space between `]` and `get;`

File: src/appkit.cs [hunk 3]
```diff
@@ -23509,7 +23553,7 @@ interface NSFilePromiseProviderDelegate
 		string GetFileNameForDestination (NSFilePromiseProvider filePromiseProvider, NSUrl destinationUrl);
 
 		[Export ("filePromiseProvider:writePromiseToURL:completionHandler:")]
-		void WritePromiseToUrl (NSFilePromiseProvider filePromiseProvider, NSUrl url, Action<NSError> completionHandler);
+		void WritePromiseToUrl (NSFilePromiseProvider filePromiseProvider, NSUrl url, [NullAllowed]Action<NSError> completionHandler);
```
Comment (spouliot) [inline src/appkit.cs]: space after `[` - broken keyboard ;-)

File: src/appkit.cs [hunk 4]
```diff
@@ -12771,8 +12793,14 @@ public interface NSSliderCell {
...
-		[Export ("isVertical")]
-		nint IsVertical { get; /* FIXME: [Mac (10, 12)] set; -- Radar 27222357 */ }
+		[Export ("vertical")]
+		// Radar 27222357
+#if XAMCORE_4_0
+		bool
+#else
+		nint
+#endif
+		IsVertical { [Bind ("isVertical")]get; [Mac (10, 12)] set; }
```
Comment (spouliot) [inline src/appkit.cs]: space after `[`

File: src/appkit.cs [hunk 5]
```diff
@@ -12680,8 +12696,14 @@ public interface NSSlider : NSAccessibilitySlider {
...
```
Comment (spouliot) [inline src/appkit.cs]: space after `[`

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by xamarin-release-manager** [comment]
Build failure

**Comment by spouliot** [comment]
2016-07-13 17:44:51.724 apitest[38958:14700904] -[NSSlider setVertical:]: unrecognized selector sent to instance 0x7f803de79830
2016-07-13 17:44:51.727 apitest[38958:14700904] An uncaught exception was raised
2016-07-13 17:44:51.727 apitest[38958:14700904] -[NSSlider setVertical:]: unrecognized selector sent to instance 0x7f803de79830

**Comment by chamons** [comment]
Yeah I forgot to run it after that change, I thought fd5446b5693f74936867bca339acca3a93428cee would be enough to fix. Let me look.

**Comment by xamarin-release-manager** [comment]
Build failure

**Comment by chamons** [comment]
@spouliot All green now except watchOS test failures. Those known?

**Comment by spouliot** [comment]
@chamons the crash on monotouch-test/watchos is random (but not the first time it happened). 
The other one will be fixed soon (PR being built). In any case they are not related to this commit.

**Pull request merged**